src/neo/SmartContract/Native/Tokens/NeoToken.cs

```diff
@@ -101,10 +94,19 @@ internal override bool Initialize(ApplicationEngine engine)
         {
             if (!base.Initialize(engine)) return false;
             if (base.TotalSupply(engine.Snapshot) != BigInteger.Zero) return false;
-            UInt160 account = Contract.CreateMultiSigRedeemScript(Blockchain.StandbyValidators.Length / 2 + 1, Blockchain.StandbyValidators).ToScriptHash();
-            Mint(engine, account, TotalAmount);
-            foreach (ECPoint pubkey in Blockchain.StandbyValidators)
-                RegisterValidator(engine.Snapshot, pubkey);
+            BigInteger amount = TotalAmount;
+            for (int i = 0; i < Blockchain.CommitteeMembersCount; i++)
+            {
+                ECPoint pubkey = Blockchain.StandbyCommittee[i];
+                RegisterCandidate(engine.Snapshot, pubkey);
+                BigInteger balance = TotalAmount / 2 / (Blockchain.ValidatorsCount * 2 + (Blockchain.CommitteeMembersCount - Blockchain.ValidatorsCount));
+                if (i < Blockchain.ValidatorsCount) balance *= 2;
+                UInt160 account = Contract.CreateSignatureRedeemScript(pubkey).ToScriptHash();
+                Mint(engine, account, balance);
+                Vote(engine.Snapshot, account, pubkey);
+                amount -= balance;
+            }
+            Mint(engine, Blockchain.GetConsensusAddress(Blockchain.StandbyValidators), amount);
```

Comment:
Is not better to mint all the neo to the multi sig?

Reply:
We need to vote for the default committee.